src/components/Toggle/__stories__/toggle.stories.mdx

```diff
@@ -0,0 +1,131 @@
+    {toggleTemplate.bind({})}
+  </Story>
+</Canvas>
+
+## Props
+<ArgsTable of={Toggle} />
+
+## Usage
+<UsageGuidelines guidelines={[
+  "Use toggles when your intent is to turn something on or off instantly.",
+  "Let users know what happens when the toggle is switched by using a tooltip.",
+  "Toggle can either be selected or not selected. They cannot be in an indeterminate state (unlike checkboxes).",
+  "Use labels such as “on” and “off” or “enable” and “disable” to communicate the state of the toggle."
+]}/>
+
+<Tip title="Tip">
+  If the user needs to choose an item from a set of options, use Radio button or Checkboxes instead.
+</Tip>
+
+## Variants
+### States
+<Canvas>
+  <Story name="States">
+    <MultipleStoryElementsWrapper className="monday-style-story-toggle_column">
+      <Toggle isDefaultSelected={false} />
+      <Toggle />
+      </MultipleStoryElementsWrapper>
+  </Story>
+</Canvas>
+
+### Disabled
+<Canvas>
+  <Story name="Disabled">
+    <MultipleStoryElementsWrapper className="monday-style-story-toggle_column">
+      <Toggle isDefaultSelected={false} isDisabled={true} />
+      <Toggle isDisabled={true} />
+    </MultipleStoryElementsWrapper>
+  </Story>
+</Canvas>
+
+
+## Do’s and Don’ts
+<ComponentRules
+  rules={[
+    {
+      positive: {
+        component:
+          <MultipleStoryElementsWrapper className="monday-style-story-toggle_row">
+            Dark mode <Toggle />
+          </MultipleStoryElementsWrapper>,
+        description:"Use toggle only for on and off actions. We recommend not to change these lables values."
+      },
+      negative: {
+        component:
+          <MultipleStoryElementsWrapper className="monday-style-story-toggle_row"> 
+            Theme <Toggle offOverrideText="Light mode" onOverrideText="Dark mode" />
+          </MultipleStoryElementsWrapper>,
+        description:"Don’t use toggle for configurations. Instead, use Checkboxes or Radio buttons."
+      }
+    },
+    {
+      positive: {
+        component:<Toggle  />,
+        description:"Toggle will always apear with lables."
+      },
+      negative: {
+        component:
+        <MultipleStoryElementsWrapper className="monday-style-story-toggle_column">
+          <Toggle offOverrideText="" /><Toggle onOverrideText="" offOverrideText="" />
+        </MultipleStoryElementsWrapper>,
+        description:"Don’t remove toggle lables, since users wouldn’t know what state it represents. "
+      }
+    },
+  ]}
+/>
+
+## Use cases and examples
+
+### Turn on/ off an automation
+In the automations center, a user can turn the automation on or off.
+<Canvas>
+  <Story name="Turn on/ off an automation">
+    <b>Board automations</b>
+    <Toggle />
+  </Story>
+</Canvas>
+
+### Turn on/ off notifications
+In the automations center, a user can turn the automation on or off.
+<Canvas>
+  <Story name="Turn on/ off an notifications">
+    <b>Do not disturb mode</b>
+    <Toggle ariaControls="asdsad" isDefaultSelected={false} />
```

Comment:
If the toggle does not control anything I think that there is not need to pass ariaControls. What do you think @orrgottlieb ?

Reply:
i agree